### PR TITLE
Highlighting reviews - documentation and readme links

### PR DESCRIPTION
**Pull request by mallewellyn**
Hi!

As Ailith and I have discussed, I've collated all the feedback from reviews and instructors & The Carpentries guidelines so that it's clear what's left to be done both to us and Carpentries reviewers. 

Hopefully this is helpful but very happy to change the format or text, whatever's useful! Also, I essentially just went through the entire repo and picked out all reviews and feedback. I could well have missed some within the repo or any stored externally, so happy to edit if you can see some are missing.

I've also edited the readme slightly to link to the reviews, but also changed some of the initial text as I think this was just a default from the initial clone? Of course, happy to revert.

Hopefully seeing the list of to dos isn't too annoying - I'll try make some good progress with these!

### CODE REVIEW

File: _extras/data.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+- Synonym:
+- Accession:
+- GID:
+- Annotation:
+- Product:
+- Marginal.Age.Relationship:
+
+  Can't find these data or a full data description in the paper? It's also not clear to me what these data record? Methylation markers by age?
+  Is the source the right reference?
+
```
Comment (alanocallaghan) [inline _extras/data.md:75]: I will see if I can remove this for simplicity, may be easier to just use the CpGs from it without loading the whole thing

Comment (mallewellyn) [reply _extras/data.md:75]: Ok! I'll try to fill after

**Comment by mallewellyn** [comment]
Have tried to create a data page as far as I can with my primitive biosciences knowledge. Will absolutely need checking and filling out in places!!

File: _extras/data.md (hunk 1 above)
Comment (ailithewing) [reply _extras/data.md:75]: Yes to ditching all apart from CpG markers!

Comment (mallewellyn) [reply _extras/data.md]: 60b9da3 ahead of changes to data set in Alan's PR #166

File: _extras/data.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  - group #: group number
+  - total mRNA mol:
+  - well:
+  - sex: sex
+  - age: age 
+  - diameter:
+  - cell_id: cell identifier 
+  - level1class:
+  - level2class:
+  - sizeFactor:
```
Comment (alanocallaghan) [inline _extras/data.md:84]:
```suggestion
  - total mRNA mol: total number of observed mRNA molecules corresponding to this cell's unique barcode identifier
    - well: the well that this cell's cDNA was stored in during processing
  - sex: sex of the donor animal
  - age: age of the donor animal 
  - diameter: estimated cell diameter
  - cell_id: cell identifier 
  - level1class: a cluster label identified using a mix of computational techniques and manual annotation
  - level2class: a cluster label identified using a mix of computational techniques and manual annotation
  - sizeFactor:   estimate size factor calculated for scaling normalisation using (e.g., **`scran`**).
```

File: reviews.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+-  Tested that the list of required prior skills and/or knowledge is complete and accurate.
+-  The setup and installation instructions are complete, accurate, and easy to follow.
+- It has been taught at least two times by Instructors who had not been heavily involved in the development of the 
+lesson before that point.
+- Check that the lesson includes exercises in a variety of formats. 
+
+&#x2612; The points still to be addressed are:
+
+- Conversion to The Carpentries Workbench
+- Alt text for Episode 3 and Challenge figures in Episode 4, 6 and 7 required.
```
Comment (alanocallaghan) [inline reviews.md:141]: Think this will be untrue?

File: reviews.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+ - The factor analysis episode could be combined into another episode (potentially doesn't need to be its own episode) since the course is pitched at biologists. This could be combined into the PCA episode in a callout.
+- In episode 2, the DNA methylation data discussion could be moved to episode 1, the broom package is possibly unnecessary and we could just use summary(), the advanced content to compute the t-statistics by hand would probably be better off in a sidebar.
+- In episode 3, the Introduction and coefficient estimates section could be moved to episode 2, the coefficient estimates section should probably go before discussion of singularities, more explanation of the heat map in the cross-validation section required to clarify what it shows.
+
+## Carpentries-specific 
+
+&#x2611; The lesson has been developed using The Carpentries template. As such, a number of requirements are fulfilled:
+
+- Alt text and captions complete for Episodes 1, 2, 5 in line with [The Carpentries guide](https://carpentries.org/blog/2022/11/accathon/).
```
Comment (alanocallaghan) [inline reviews.md:120]: As below, this is untrue now I think?

**Comment by alanocallaghan** [comment]
Looks good to me but will leave the final word for @ailithewing.

I think re:

> - The lesson does not make use of superfluous data sets: ensuring that later uses of microarray and scrnaseq data is justified (as opposed to prostate/methylation).

that ideally one could probably just use the methylation data for everything without much loss, as long as the results aren't totally contrived. This might reduce the cognitive load a bit from hopping dataset to dataset every couple of hours. However it's a bit of a stretch goal right now to say the least as it would require a hell of a lot of rewriting

File: reviews.md (hunk 3 above)
Comment (mallewellyn) [reply reviews.md]: Updated in 83fa691

File: reviews.md (hunk 4 above)
Comment (mallewellyn) [reply reviews.md]: Updated in 83fa691

**Comment by mallewellyn** [comment]
Agreed, I think using the methylation data would be good. However, on the whole, I don't think the current data sets are 'superfluous' in that it's probably also good to strike a balance between seeing how these methods behave on different examples and cognitive load. There are some small examples where another data set is introduced where it probably doesn't need to be though, e.g. the use of cars, which also feels a bit random from a biosciences perspective. I'll take another look to see if I can propose minor changes that would sort this.

**Comment by mallewellyn** [comment]
Also, thank you for blitzing the other PRs! Feels like really good progress!!

**Comment by alanocallaghan** [comment]
cars is in episode 2, in which we've already created a linear model object, so should just call model.matrix on that instead of making a new one

File: CITATION [hunk 5]
```diff
@@ -1 +1,2 @@
-FIXME: describe how to cite this lesson.
\ No newline at end of file
+O’Callaghan, A., Robertson, G., Vallejos, C., Ewing, A., Meynert, A., and Becher, H. (2024). High dimensional statistics with R. https://github.com/
```
Comment (ailithewing) [inline CITATION:1]:
```suggestion
O’Callaghan A, Robertson G, LLewellyn M, Becher H, Meynert A, Vallejos C, Ewing A. (2024). High dimensional statistics with R. https://github.com/
```

**Review by ailithewing — APPROVED** [review]

**Review by ailithewing — COMMENTED** [review]
These all look great. I think it's worth keeping the small number of datasets we have rather than using methylation for all. It illustrates the generalisability in approach and also gives biologists (our target audience) more applications to relate to without being unmanageable.

**Pull request merged**